diagnostic.md

```diff
@@ -21,13 +21,18 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --include --request "GET" "http://www.example.com"
```

Comment:
Don't forget to get what you want back from "http://www.example.com". so it would be:
```sh
curl --include --request "GET" "http://www.example.com/donuts/"
```